h/static/styles/group-form.scss

```diff
@@ -105,6 +105,11 @@
 
     &__list li {
       margin-bottom: 1.5em;
+
+      em {
```

Comment:
Edit: Ignore the above, I see we're using just `<em>` to avoid needing to know where the link will be used when it is generated.